Tests/KituraWebSocketTests/KituraTest.swift

```diff
@@ -105,6 +105,12 @@ class KituraTest: XCTestCase {
         WebSocket.register(service: service, onPath: onPath ?? servicePath)
     }
     
+    func register(onPath: String? = nil, closeReason: WebSocketCloseReasonCode, testServerRequest: Bool = false, pingMessage: String? = nil, connectionTimeout: Int? = nil) -> TestWebSocketService {
```

Comment:
Do we need this extra function? Could we make the `register()` function above return a service instead?